### PR TITLE
product-cards

### PR DESCRIPTION
**Pull request by SichVictor**
https://sichvictor.github.io/layout_product-cards/
https://sichvictor.github.io/layout_product-cards/report/html_report/

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -5,9 +5,42 @@
...
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./styles/index.scss">
   </head>
-  <body>
-    <h1>Product cards</h1>
+  <body class="body">
+
+    <div class="card" data-qa="card">
+      <img
```
Comment (mykhalenych) [inline src/index.html:16]: Screenshot 2023-06-01 at 17 21 15

File: src/styles/index.scss [hunk 2]
```diff
@@ -1,3 +1,119 @@
...
+    font-weight: 500;
+    line-height: 18px;
+    color: $color--text;
+    margin: 36px 16px 4px;
+  }
+
+  &__code {
+    display: flex;
+    font-size: 10px;
+    font-weight: 400;
```
Comment (mykhalenych) [inline src/styles/index.scss:34]: it is by default

File: src/styles/index.scss [hunk 3]
```diff
@@ -1,3 +1,119 @@
...
+  font-family: Roboto, sans-serif;
   margin: 0;
 }
+
+.card {
+  width: 200px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  box-sizing: border-box;
```
Comment (mykhalenych) [inline src/styles/index.scss:15]: move to `*` selector

File: src/styles/index.scss [hunk 4]
```diff
@@ -1,3 +1,119 @@
...
+    &:hover {
+      background-color: #fff;
+      color: $color--button;
+      border: 1px solid $color--button;
+      border-radius: 5px;
+    }
+  }
+}
+
+.stars {
```
Comment (mykhalenych) [inline src/styles/index.scss:99]: create stars.scss with stars styles, card.scss with card etc

**Review by mykhalenych — CHANGES_REQUESTED** [review]

File: src/index.html [hunk 5]
```diff
@@ -5,9 +5,45 @@
...
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./styles/index.scss">
   </head>
-  <body>
-    <h1>Product cards</h1>
+  <body class="product-cards">
+
+    <div
```
Comment (polosanya) [inline src/index.html:15]:
```suggestion
  <body class="product-cards">
    <div
```
Don't add empty lines between parent and child elements

File: src/index.html (hunk 1 above)
Comment (polosanya) [reply src/index.html:19]: Not fixed

File: src/index.html [hunk 6]
```diff
@@ -5,9 +5,45 @@
...
+      <img
+      class="card__image"
+      src="../src/images/imac.jpeg"
+      alt="Photo Product"
+      >
+
+      <h3 class="card__title">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h3>
+      <span class="card__code">Product code: 195434</span>
+
+      <div class="card__reting">
```
Comment (polosanya) [inline src/index.html:28]:
```suggestion
      <div class="card__rating">
```

File: src/styles/index.scss [hunk 7]
```diff
@@ -1,3 +1,98 @@
...
+  box-sizing: border-box;
+}
+
+.product-cards {
+  font-family: Roboto, sans-serif;
   margin: 0;
 }
+
+.card {
+  width: 200px;
```
Comment (polosanya) [inline src/styles/index.scss:17]: It would be great to create variables for all main values (main sizes, colors etc)

File: src/styles/stars.scss [hunk 8]
```diff
@@ -0,0 +1,21 @@
...
+    background-repeat: no-repeat;
+    background-position: center;
+  }
+
+  &__star:not(:first-child) {
+    margin-left: 4px;
+  }
+
+  &--0 &__star:nth-child(-n + 4) {
+    background-image: url(/images/star-active.svg);
```
Comment (polosanya) [inline src/styles/stars.scss:19]: Stars should display the correct amount of active stars according to the modifier
image

File: src/styles/stars.scss [hunk 9]
```diff
@@ -0,0 +1,21 @@
+.stars {
+  display: flex;
+  margin-left: 16px;
+  box-sizing: border-box;
```
Comment (polosanya) [inline src/styles/stars.scss:4]: You've already set box-sizing border-box for all elements. No need to specify it here

**Review by polosanya — CHANGES_REQUESTED** [review]
Almost done!

File: src/index.html (hunk 5 above)
Comment (SichVictor) [reply src/index.html]: imageТут я вніс зміни ще попереднього разу незнаю чому не відображаються

File: src/index.html [hunk 10]
```diff
@@ -5,9 +5,43 @@
...
-  <body>
-    <h1>Product cards</h1>
+  <body class="product-cards">
+    <div
+      class="card"
+      data-qa="card"
+    >
+      <img
+      class="card__image"
+      src="../src/images/imac.jpeg"
```
Comment (etojeDenys) [inline src/index.html:20]: The 'src' attribute of the 'img' tag should use the correct path for the image file. Change the path to './images/imac.jpeg'.

File: src/index.html (hunk 1 above)
Comment (etojeDenys) [reply src/index.html:18]: still  not fixed

**Review by etojeDenys — CHANGES_REQUESTED** [review]

**Review by VitaliyBondarenko1982 — APPROVED** [review]
Looks good to me

**Review by etojeDenys — APPROVED** [review]

**Pull request closed without merging**